test/post-911-gib-status/components/EnrollmentHistory.unit.spec.jsx

```diff
@@ -52,12 +53,15 @@ describe('<EnrollmentHistory>', () => {
     expect(featureBoxes[0].text()).to.equal('Does something look wrong in your enrollment history?');
   });
 
-  it('should show history may be incorrect warning if benefit pending', () => {
+  it('should show history may be incorrect warning if not active duty', () => {
```

Comment:
It looks like we no longer condition the history warning on `activeDuty`, right?